### PR TITLE
Αdded icon in network quality warning

### PR DESCRIPTION
**Pull request by tritonas00**
Just an icon, nothing fancy.

Before:
kkk
After:
kk

### CODE REVIEW

File: source/main/gui/panels/GUI_MultiplayerClientList.cpp [hunk 1]
```diff
@@ -65,6 +65,7 @@ void MpClientList::Draw()
 
     int y = 20 + (ImGui::GetTextLineHeightWithSpacing() * m_users.size());
 
+    Ogre::TexturePtr warn_tex = FetchIcon("error.png");
```
[the remark this answers is not in the document]
Comment (tritonas00) [reply source/main/gui/panels/GUI_MultiplayerClientList.cpp:68]: Why fetch it every time quality is low? I just went by `fetch it once and draw it when we want`

After coding the repo panel i acquired a post-traumatic psychosis with those things :laughing:

[the remark this answers is not in the document]
Comment (only-a-ptr) [reply source/main/gui/panels/GUI_MultiplayerClientList.cpp:68]: Ah, I see, except this code doesn't accomplish your goal. I'll push a commit which completes it.

[the remark this answers is not in the document]
Comment (tritonas00) [reply source/main/gui/panels/GUI_MultiplayerClientList.cpp:68]: Its just an icon, no big deal i think, but thanks please do it properly.

**Review by only-a-ptr — APPROVED** [review]
The icon is a nice touch, thanks.

**Pull request merged**